Feature/update goamz aws (#65)
ONSdigital/dp-deployer: README.md

    ```

5. Run `make deployment` and this should build an image for your new updated code, push the image to `ECR` and bundle it to s3.
    **Note:** The tar bundle which includes a nomad plan can be seen in s3 which is always under production/ no matter which environment ansible is targetting. The nomad plan points to the ECR image.
6. Go to `dp-setup` and check you are in the right environment to run ansible. It is recommended you stick with `sandbox` for testing. Amend the `dp_deployer_version` from the output of the `make deployment` command.

    ```bash
    vim +/dp_deployer_version dp-setup/ansible/roles/bootstrap-deployer/defaults/main.yml
    ```

7. After updating the `dp_deployer_version`, run the ansible-playbook command to bootstrap the deployer.

    ```bash
    export ONS_DP_ENV = sandbox
    ansible-playbook --vault-id=$(ONS_DP_ENV)@.$(ONS_DP_ENV).pass -i inventories/$(ONS_DP_ENV) bootstrap-deployer.yml
    ```

8. Check [nomad-ui](https://nomad.dp.aws.onsdigital.uk/ui/jobs/dp-deployer/versions) if the deployer has been deployed successfully.

> 9. Go to [concourse-ui](https://concourse.dp-ci.aws.onsdigital.uk/) and deploy the `dp-import-reporter` and then trigger `env-ship-it` to test the deployer code.

Reviewer: ```suggestion
9. Go to [concourse-ui](https://concourse.dp-ci.aws.onsdigital.uk/) and deploy the `dp-import-reporter` and then trigger `<env>-ship-it` to test the deployer code.
```

> 10. If the previous step has been successful, trigger the secrets pipeline to confirm that it is working as expected.

Reviewer: ```suggestion
10. If the previous step has been successful, trigger the `secrets` pipeline to confirm that it is working as expected.
```

> 11. If it hasn't been successful, roll back to the previous version of the deployer, by updating the `dp-deployer-version` in `dp-setup` as mentioned in step 6 and run the `ansible-playbook bootstrap-deployer` command as shown in step 7.

Reviewer: ```suggestion
11. If it hasn't been successful, rollback to the previous version of the deployer, by reverting the `dp_deployer_version` in `dp-setup` as mentioned in step 6 and then re-apply the `bootstrap-deployer` playbook as shown in step 7.
```

### Licence

> Copyright © 2019, Office for National Statistics (https://www.ons.gov.uk)

Reviewer: ```suggestion
Copyright © 2025, Office for National Statistics (https://www.ons.gov.uk)
```
